content/en/blog/_posts/2021-04-12-introducing-suspended-jobs.md

```diff
@@ -55,17 +55,17 @@ spec:
 ```
 
 Jobs are not suspended by default, so I'm explicitly setting the `suspend` field
-to true in the above Job spec. In the above example, the Job controller will
+to _true_ in the above Job spec. In the above example, the Job controller will
```

Comment:
I prefer to keep "Job spec" here to emphasise that it's in the `.spec` in addition to colloquially referring to the YAML. If you feel strongly about "Job manifest", I can change.

Reply:
How about “the `.spec` of the Job manifest?”

Reply:
sounds good, updated.